### ISSUE TITLE
Sorting order on DetailsList column click is not announced on Edge + Narrator, Edge + NVDA, Chrome + Narrator

### ISSUE THREAD

**Issue by nkalpana** [OfficeDev/office-ui-fabric-react#11297]
### Environment Information

- **Package version(s)**: Latest version of React (used an example's export to Codepen code)
- **Browser and OS versions**: Edge + Narrator, Chrome + Narrator

### Describe the issue:

On a simple DetailsList example, (exported to codepen), I notice that the column sorting order is being announced on column click when the sortAscending/sortDescendingAriaLabel parameters are set on the IColumn, when we use Chrome + NVDA, Edge + NVDA.
However this does not work on Chrome + Narrator, or Edge + Narrator

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/nkalpana/pen/QWWRaEz?editors=0010
All I did was take the first example of DetailsList in controls help guide, and export to code pen. I commented out the code that sets the <Announcer ../> control's message when column's sorting order has changed, so that the code merely uses the sortAscending/sortDescendingAriaLabel values in each IColumn def of the Name column (lines 107, 108)

#### Actual behavior:
Sorting order is not announced as per the sortAscending/DescendingAriaLabel parameters, when using Edge + Narrator, or Chrome + Narrator. Works as expected in Chrome + NVDA.
but does not work for:
- Edge + NVDA
- Edge + Narrator
- Chrome + Narrator

#### Expected behavior:
Sorting order should be announced as per the sortAscending/DescendingAriaLabel values in all Chrome/Edge, and NVDA/Narrator combinations

**Comment by khmakoto** [comment]
Hi @nkalpana, thanks for filing this issue! I wonder why you commented out the `Announced` control that made this work. This behavior is by design with the use of Narrator given that the decision to announce the change of an `aria-label` is a decision that the screen reader takes. Narrator does not do that, which is why the example has the `Announced` part so that the change in sorting is announced by the screen reader.

**Comment by nkalpana** [comment]
Thanks for the update. 
It will be great if you can enable this for Narrator without us having to use the Annouced control.
It will also be great if you can include accessibility info like this with the controls' info in controls playground.
